docs/pages/database-access/reference/aws.mdx

```diff
@@ -3,11 +3,14 @@ title: Database Access AWS IAM Reference
 description: AWS IAM policies for Teleport database access.
 ---
 
-Teleport automatically discovers and configures IAM for RDS and Aurora access
-given proper IAM permissions as described in the [RDS guide](../guides/rds.mdx).
+Teleport automatically discovers and configures IAM for RDS and Redshift access
+given proper IAM permissions as described in the [AWS RDS & Aurora
+guide](../guides/rds.mdx) and the [AWS Redshift
+guide](../guides/postgres-redshift.mdx).
 
-If you prefer to register RDS databases manually using [static configuration](./configuration.mdx)
-or [`tctl`](../guides/dynamic-registration.mdx) and manage IAM yourself, example
+If you prefer to register RDS or Redshift databases manually using a [static
+configuration](./configuration.mdx) or
+[`tctl`](../guides/dynamic-registration.mdx) and manage IAM yourself, example
```

Comment:
I think the phrase "example IAM policies you'd need to create" is potentially misleading. Do readers managing IAM themselves need to create each of these policy resources, but can modify the contents of each one to suit their needs? Or something else? Or does this phrase mean, "example IAM policies you can create"?

Reply:
yes. it's more like here are example policies that show what permissions you need to have in your own managed IAM policies. will update